backend/src/types/DummyResponse.ts

```diff
@@ -0,0 +1,22 @@
+import { Prompt, Gpt3Response } from '../types';
+
+/**
+ * Debug info that the dummy provides, but an actual
+ * response from gpt3 would not contain
+ */
+interface DebugInfo {
```

Comment:
I'd write all of these like `export interface DebugInfo {...}` instead of `interface DebugInfo {...}; export { DebugInfo }`.

Reply:
Types should now use `export interface` consistantly.